mne/viz/ica.py

```diff
@@ -130,6 +131,159 @@ def plot_ica_sources(ica, inst, picks=None, exclude=None, start=None,
     return fig
 
 
+def _ica_properties_skeleton():
```

Comment:
Name isn't really descriptive IMO. Can you at least add a one-line docstring of what this does?